docs/integrations/index.rst

```diff
@@ -40,6 +41,14 @@ The functions below facilitate the process of collecting the information needed.
 
 .. autofunction:: polaris.integrations.WithdrawalIntegration.after_form_validation
 
+Some wallets may pass fields documented in SEP-9_ to the `/interactive` endpoints for
```

Comment:
`to the /interactive endpoints` makes it sound like you're passing it to the webapp.  Maybe say `to the initial POST request`